compiler/aarch64/codegen/OMRMachine.hpp

```diff
@@ -62,14 +62,30 @@ class OMR_EXTENSIBLE Machine : public OMR::Machine
     */
    Machine(TR::CodeGenerator *cg);
 
+   /**
+    * @brief Finds the best free register
+    * @param[in] currentInstruction : current instruction
+    * @param[in] rk : register kind
+    * @param[in] considerUnlatched : consider unlatched state or not
+    * @param[in] virtualReg : virtual register
+    * @return Free RealRegister
+    */
+   TR::RealRegister *findBestFreeRegister(TR::Instruction *currentInstruction,
+                                            TR_RegisterKinds rk,
+                                            bool considerUnlatched = false,
+                                            TR::Register *virtualReg = NULL);
+
    /**
     * @brief Finds the best free register
     * @param[in] rk : register kind
     * @param[in] considerUnlatched : consider unlatched state or not
     * @return Free RealRegister
     */
    TR::RealRegister *findBestFreeRegister(TR_RegisterKinds rk,
-                                            bool considerUnlatched = false);
+                                            bool considerUnlatched = false)
+      {
+      return findBestFreeRegister(NULL, rk, false);
```

Comment:
Is it OK to pass `false` as the third argument here?
I guess it should be `considerUnlatched` instead.

Reply:
Thanks for pointing it out. Fixed.